doc/developer/design.md

```diff
@@ -168,4 +168,30 @@ The permissions are different on Unix and on Windows. So, the `executable`
 field is just ignored on Windows.
 
 
+Local watcher
+-------------
+
+Historically, the local watcher was based on the chokidar library. Chokidar is
+a high-level library that tries to mask the differences between the API for
+watching files on 3 OSes: ReadDirectoryChangesW on Windows, FsEvents on macOS,
+and inotify on Linux. But, we discover that doing so leads to some pitfalls,
+for our use cases. In particular, on Windows, Chokidar works with polling or
+with creating a new watcher per directory, while a single watcher on the sync
```

Comment:
`s/with creating/by creating`

Reply:
"with the creation" or "with creating" or "by creating" are all correct, according to deepl:

https://www.deepl.com/translator#fr/en/En%20particulier%2C%20sous%20Windows%2C%20Chokidar%20travaille%20avec%20polling%20ou%20avec%20la%20cr%C3%A9ation%20d'un%20nouveau%20watcher%20par%20r%C3%A9pertoire